src/WebJobs.Script.WebHost/App_Start/WebApiConfig.cs

```diff
@@ -114,10 +114,11 @@ private static WebHostSettings GetDefaultSettings()
             WebHostSettings settings = new WebHostSettings();
 
             string home = Environment.GetEnvironmentVariable("HOME");
+            string scriptPath = Environment.GetEnvironmentVariable("SCRIPTPATH");
```

Comment:
@mathewc not really. looks like the approach in the wiki only works for the console app(WebJobs.Scriot.Host) but not if we want to run the web app(WebJobs.Script.WebHost) locally.